src/test/java/mops/termine2/scheduling/GruppeSchedulerTest.java

```diff
@@ -1,194 +1,194 @@
-		
-		server.expect(ExpectedCount.once(),
-			requestTo("http://localhost:8082/gruppen2/api/updateGroups/0"))
-			.andExpect(method(HttpMethod.GET))
-			.andRespond(withSuccess(groupNew2, MediaType.APPLICATION_JSON));
-		
-		scheduler.updateGruppe();
-		
-		verify(bgrepo, times(1)).deleteByBenutzerAndGruppeId("string2", 0L);
-		verify(bgrepo, times(1)).save(erwartetNeu);
-	}
-	
-	@Test
-	public void aendereGruppenname() {
-		when(bgrepo.findBenutzerByGruppeId(0L))
-			.thenReturn(new ArrayList<String>());
-		when(bgrepo.findGruppeByGruppeId(0L))
-			.thenReturn(Optional.of("string"));
-		BenutzerGruppeDB erwartetNeu = new BenutzerGruppeDB();
-		erwartetNeu.setBenutzer("string1");
-		erwartetNeu.setGruppe("stringNeu");
-		erwartetNeu.setGruppeId(0L);
-		
-		server.expect(ExpectedCount.once(),
-			requestTo("http://localhost:8082/gruppen2/api/updateGroups/0"))
-			.andExpect(method(HttpMethod.GET))
-			.andRespond(withSuccess(groupNew3, MediaType.APPLICATION_JSON));
-		
-		scheduler.updateGruppe();
-		
-		verify(bgrepo, times(1)).deleteAllByGruppeId(0L);
-		verify(bgrepo, times(1)).save(erwartetNeu);
-	}
-	
-	@Test
-	public void keineAenderung() {
-		server.expect(ExpectedCount.once(),
-			requestTo("http://localhost:8082/gruppen2/api/updateGroups/0"))
-			.andExpect(method(HttpMethod.GET))
-			.andRespond(withSuccess(groupNoChange, MediaType.APPLICATION_JSON));
-		
-		scheduler.updateGruppe();
-		
-		verify(bgrepo, never()).deleteAllByGruppeId(any());
-		verify(bgrepo, never()).save(any());
-		verify(bgrepo, never()).findGruppeByGruppeId(any());
-		verify(bgrepo, never()).findBenutzerByGruppeId(any());
-		verify(bgrepo, never()).deleteByBenutzerAndGruppeId(any(), any());
-	}
-	
-}
+package mops.termine2.scheduling;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Optional;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.client.ExpectedCount;
+import org.springframework.test.web.client.MockRestServiceServer;
+import org.springframework.web.client.RestTemplate;
+
+import mops.termine2.database.BenutzerGruppeRepository;
+import mops.termine2.database.entities.BenutzerGruppeDB;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.client.match.MockRestRequestMatchers.method;
+import static org.springframework.test.web.client.match.MockRestRequestMatchers.requestTo;
+import static org.springframework.test.web.client.response.MockRestResponseCreators.withSuccess;
+
+public class GruppeSchedulerTest {
+	
+	private static String groupNew1 = "{ \"groupList\": [ { \"description\": \"string\", \"id\": 0, \"members\": "

```

Comment:
Bitte in den Teststrings jeweils die `0` bei `id` zu `\"0\"` ändern, damit das auch auf jeden Fall als String gewertet wird, und nicht als Zahl, die dann zu einem String umgewandelt wird.